crates/vm/backends/levm/db.rs

```diff
@@ -35,17 +40,25 @@ impl LevmDatabase for StoreWrapper {
         acc_info.is_some()
     }
 
-    fn get_storage_slot(&self, address: CoreAddress, key: CoreH256) -> CoreU256 {
-        self.store
+    fn get_storage_slot(
+        &self,
+        address: CoreAddress,
+        key: CoreH256,
+    ) -> Result<ethrex_common::U256, DatabaseError> {
+        Ok(self
+            .store
             .get_storage_at_hash(self.block_hash, address, key)
-            .unwrap()
-            .unwrap_or_default()
+            .map_err(|e| DatabaseError::Custom(e.to_string()))?
+            .unwrap_or_default())
     }
 
-    fn get_block_hash(&self, block_number: u64) -> Option<CoreH256> {
-        let a = self.store.get_block_header(block_number).unwrap();
+    fn get_block_hash(&self, block_number: u64) -> Result<Option<CoreH256>, DatabaseError> {
+        let a = self
+            .store
+            .get_block_header(block_number)
+            .map_err(|e| DatabaseError::Custom(e.to_string()))?;
 
-        a.map(|a| CoreH256::from(a.compute_block_hash().0))
+        Ok(a.map(|a| CoreH256::from(a.compute_block_hash().0)))
```

Comment:
I think something like this could be done. Else I'd change the `a` name to something more descriptive
```rs
        Ok(self
            .store
            .get_block_header(block_number)
            .map_err(|e| DatabaseError::Custom(e.to_string()))?
            .map(|header| CoreH256::from(header.compute_block_hash().0)))
```